src/components/templates/Auditorium/components/SectionPreview/SectionPreview.scss

```diff
@@ -75,3 +75,7 @@ $avatar-size: 20px;
     width: $avatar-size;
   }
 }
+
+.VenueWithOverlay {
+  margin: -($spacing--md);
+}
```

Comment:
this style is applied all over the project, so the margin is applied even on the places where it's not supposed to be.

Reply:
moved it inside the component styles fixed 65710bd